### DISCUSSION TITLE
V3 Hardware - No Screen Output

### DISCUSSION THREAD

**Discussion by SolidConqueror** [sanni/cartreader#325] [Need help]
Hey~
I built the V3 of the hardware and have tried SW V6.6 and SW7.3.
I have traced the pins to the screen and I have PIN1 tied to GND and PIN2 to VCC.

I measure the pins to be PIN2 ~4.8, PIN3 3.6, PIN4 3.7 in relation to GND PIN1.

My Arduino sketch uploads with no issues.
When I reboot with the SD card inserted I get a BLUE LED.
I can change the color of the LED by pressing the 2 buttons in various orders.

I can change it to GREEN, BLUE, DARK BLUE, RED, MAGENTA.

I have ensured both SW versions have #define enable_OLED uncommented in options.h.

Any help to recover this project would be helpful :)
Thank You

**Comment by sanni** [comment]
Probably a dead screen, happens a lot. One reason I moved to the bigger LCD module.

**Reply by SolidConqueror** [reply]
Bummer. Thanks.
I guess I will be ordering a couple more screen hoping to get a good one.

Thanks for the great design and support!
